src/auth/routes.clj

```diff
@@ -2,19 +2,51 @@
   (:require [common-labsoft.pedestal.interceptors.auth :as int-auth]
             [common-labsoft.pedestal.interceptors.error :as int-err]
             [common-labsoft.pedestal.interceptors.schema :as int-schema]
+            [auth.adapters.token :as adapters.token]
+            [auth.controllers.user :as controllers.user]
+            [auth.controllers.token :as controllers.token]
             [io.pedestal.http.route.definition :refer [defroutes]]
             [io.pedestal.http :as http]
-            [io.pedestal.http.body-params :as body-params]))
+            [io.pedestal.http.body-params :as body-params]
+            [auth.wire.register :as wire.register]
+            [auth.wire.auth :as wire.auth]
+            [auth.wire.token :as wire.token]
+            [auth.models.user :as models.user]))
 
-(defn hello-world
-  [request]
+(defn register-user
+  [{{:keys [sqs datomic crypto]} :components register :data}]
   {:status 200
-   :body   {:res "Hello, World!"}})
+   :schema models.user/User
+   :body   (controllers.user/register-new-user! register crypto sqs datomic)})
```

Comment:
We should return the user token here, since most of the times the user will want to sign in after registering his account

Reply:
Should we return as a normal token `:token/jwt "Bearer ..."` or as custom response `:user/token ""`?